community-spotlight-conference-backend-world-affairs-conference.mdx

```diff
@@ -0,0 +1,92 @@
+
+## Modernizing a dated tech stack
+
+Samuel was tasked with choosing a tech stack for the World Affairs Conference that was meant to last the next decade. While there’s been some changes to the tech stack since the inception of the conference in the 80’s, they were overdue for an upgrade.
+
+![World Affairs Conference landing page](/images/cs-wac-website.png)
+
+For the backend, there were a few very specific requirements for different parts of the registration flows, administrative dashboards and general budget. The backend needed to:
+
+- Be able to host images
+- Speak SQL and support relational data models
+- Be a fully managed and hosted solution
+- Be cost effective and fall within the constraints of the conference budget
+
+Xata checked all of the boxes and more for their next generation tech stack.
+
+> I think we used pretty much every feature Xata offers. Our user profile and speaker images are stored in Xata, we used the summarize API for sign up and registration time series charts, preview databases with Netlify, full text search for our application and more. Having one service to provide all of this functionality made it a pretty easy decision to choose Xata.\
+> _Samuel Martineau - Title_
+
+On the frontend, [Sveltekit](https://kit.svelte.dev/), [three.js](https://threejs.org/) and [AG Grid](https://ag-grid.com/) were used to build the website, registration application and internal administrative dashboards. All of this was deployed on [Netlify](https://www.netlify.com/), where the [Xata - Netlify](https://xata.io/docs/integrations/netlify) integration was used to provide preview deployments in their CI/CD pipeline. Pull request generated previews were critical to the development process for the team, making it easy to test changes before deploying to production.
+
+## A registration flow with optimized session assignment
+
+As registration flows should be, this one was pretty simple overall and leveraged [Lucia Auth](https://lucia-auth.com/) to provide a secure onboarding experience. The form intakes your personal details, as well as lunch preferences and plenary sessions you’re most interested in attending. On submit, the data is then posted to the Xata database and the user is prompted with a security code to confirm their email and identity.
+
+![Input form for conference registration](/images/cs-wac-registration-form.png)
```

Comment:
Maybe checkin with Samuel if he is okay if his email is public or maybe blur it?

Reply:
I will be sending clean images. Also, SvelteKit is usually spelled with a capital K.

Reply:
I actually knew that and am surprised I didn't capitalize it haha. I updated the text, will wait on images from @Samuel-Martineau

Reply:
@Samuel-Martineau did you ever resend clean images or if not can you please add into this PR this morning so we can publish today? Thanks :)